src/components/loginForm/form.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import styles, { makeStyles } from "@material-ui/styles";
+import TextField from "@material-ui/core/TextField";
+import Button from '@material-ui/core/Button';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import LockIcon from '@material-ui/icons/Lock';
+const useStyles = makeStyles({
+  formContainer: {
```

Comment:
You can use `Grid` component to do flex layouts
https://material-ui.com/components/grid/#grid